### PR TITLE
Vrfs.yaml and default updates tests-integration folder

### PR DESCRIPTION
**Pull request by juburnet**
Found the issues in vrfs.yaml based on the robot tests.  
Went ahead and updated the copy of defaults (main.yaml) for the test_create folder.  We can later change this in a script or something, to keep an identical copy or link to the main default file in validate folder.

### CODE REVIEW

File: tests/integration/roles/test_create/defaults/main.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ defaults:
             enable_bpdu_guard: true
           topology_switch_trunk_po_interface:
             description: 'NetAsCode Trunk PO Interface'
-            mtu: jumbo
+            mtu: jumbo
```
Comment (mtarking) [inline tests/integration/roles/test_create/defaults/main.yml:70]: Get rid of what looks like trailing whitespace

**Review by mtarking — APPROVED** [review]

File: tests/integration/roles/test_create/defaults/main.yml [hunk 2]
```diff
@@ -67,7 +67,7 @@ defaults:
             enable_bpdu_guard: true
           topology_switch_trunk_po_interface:
```
Comment (mikewiebe) [inline tests/integration/roles/test_create/defaults/main.yml:68]: @juburnet can you check to see if we even need this defaults file?  What happens if we run the tests without it?

File: tests/integration/roles/test_create/defaults/main.yml (hunk 1 above)
Comment (juburnet) [reply tests/integration/roles/test_create/defaults/main.yml:70]: will do.  This was an exact copy of the current defaults main.yaml.  I'll look at both files and remove any extra spaces.

Comment (mtarking) [reply tests/integration/roles/test_create/defaults/main.yml:70]: I fixed the trailing whitespace in the default main.yml in the validate role

File: tests/integration/roles/test_create/defaults/main.yml (hunk 2 above)
Comment (juburnet) [reply tests/integration/roles/test_create/defaults/main.yml:68]: @mikewiebe - I was able to test more extensively today and we don't need the defaults file in the test_create folder.  When it exists in test_create, it does use it and it only uses this (it takes full priority over the validate defaults, and validate defaults is never referenced).    But when the defaults don't exist in test_create it uses the validate defaults.

We can remove the defaults folder and main.yml in the test_create folder.
I'll edit this PR

**Review by rrahimm — APPROVED** [review]

**Pull request merged**